src/Enemy/EnemyStateRunAway.cpp

```diff
@@ -0,0 +1,34 @@
+#include "Enemy/EnemyStateRunAway.h"
+#include <al/Library/LiveActor/ActorPoseKeeper.h>
+#include <al/Library/Math/MathLengthUtil.h>
+#include <al/Library/Nerve/NerveSetupUtil.h>
+#include <al/Library/Nerve/NerveUtil.h>
+
+namespace {
+NERVE_IMPL(EnemyStateRunAway, Run);
+NERVE_IMPL(EnemyStateRunAway, PanicRun);
+NERVE_IMPL(EnemyStateRunAway, PanicRunCollided);
+
+NERVE_MAKE(EnemyStateRunAway, Run);
+NERVE_MAKE(EnemyStateRunAway, PanicRun);
+NERVE_MAKE(EnemyStateRunAway, PanicRunCollided);
+}  // namespace
+
+// NON_MATCHING: Mismatching Vector math, should be equivalent
```

Comment:
decomp.me?

Reply:
Created one. We'll see what happens.

Reply:
https://decomp.me/scratch/QCGzg

Reply:
decomp.me shows that this does not match but tools/check ignores the different ldr (good behavior). Glad there's a match for this one